plugins/exporter.koplugin/template/md.lua

```diff
@@ -38,23 +42,22 @@ local formatters = {
 local function prepareBookContent(book, formatting_options, highlight_formatting)
     local tbl = {}
     local current_chapter = nil
-    table.insert(tbl, "# " .. book.title)
     local author = book.author or _("N/A")
-    table.insert(tbl, "##### " .. author:gsub("\n", ", ") .. "\n")
+    table.insert(tbl, "# " .. book.title .. "( by " .. author:gsub("\n", ", ") .. ")\n")
```

Comment:
This introduces English words. If that's desirable it should be localized. I don't know if there might be any issues with that though.

Reply:
Good point.

I’ll change that to be language-agnostic.

If it turns out that is confusing, we can change it back and use gettext to localise the string. In that case, we could also localise the timestamp.

Reply:
I changed it for a long- or em-dash, which seems to be the closest to an “common authorship character” as it gets:
https://en.wikipedia.org/wiki/Dash#Attribution_of_quote_source